source/Junkership.js

```diff
@@ -83,9 +88,10 @@ export default class Junkership extends Pixi.Sprite {
             }
         }
     }
-    
+
     onCollision(collidedWith) {
         game.removeChild(this)
+        Reference.ControlScheme.keys[this.controls].inUse = false
```

Comment:
So why are we tracking whether a control scheme is in use or not? So we don't listen for them anymore I guess?